test/functional/admin/legacy_publications_controller_test.rb

```diff
@@ -83,7 +83,7 @@ class Admin::LegacyPublicationsControllerTest < ActionController::TestCase
 
   test "should validate previously_published field on create" do
     post :create, params: { edition: controller_attributes_for(:publication).except(:previously_published) }
-    assert_equal "You must specify whether the document has been published before", assigns(:edition).errors.full_messages.last
+    assert_equal "Previously published You must specify whether the document has been published before", assigns(:edition).errors.full_messages.last
```

Comment:
This is a bit rubbish 😢 although I don't know of an obvious way around it while maintaining the hyperlink in the error summary.